tsdb/engine/tsm1/string_test.go

```diff
@@ -63,7 +63,7 @@ func Test_StringEncoder_Multi_Compressed(t *testing.T) {
 		t.Fatalf("unexpected encoding: got %v, exp %v", b[0], stringCompressedSnappy)
 	}
 
-	if exp := 47; len(b) != exp {
+	if exp := 51; len(b) != exp {
```

Comment:
That would make the test less brittle (still brittle, just less).  I'm fine with it.  @jwilder  thoughts?